internal/host/statemachine.go

```diff
@@ -298,6 +298,19 @@ func NewHostStateMachine(sm stateswitch.StateMachine, th *transitionHandler) sta
 		PostTransition:   th.PostRefreshReclaimTimeout,
 	})
 
+	// Noop transitions for refresh while reclaim has not timed out
```

Comment:
to avoid errors i suggest to move noop transitions to the end of the file
the what state machine work is going one by one over the transitions and it will stop after the first one that it catch. 
If someone will add more logic or the source state to other refresh transition it will probably be a bug